### PR TITLE
Matlab bindings v2.5.0.0

### PR DESCRIPTION
**Pull request by erikscheurer**
Release for [preCICE v2.5.0](https://github.com/precice/precice/releases/tag/v2.5.0)

Main features:
- Automated testing of solverdummies and tutorials #16 
- Remove unnecessary warning for absolute path #39

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+description: 'Installs preCICE and builds the preCICE Matlab bindings in the directory. Ends in `$GITHUB_WORKSPACE/matlab-bindings`'
+
+inputs:
+  matlab-release:
+    required: true
+    default: 'latest'
+    description: 'The release of Matlab to use.'
+  precice-version:
+    required: true
+    default: '2.4.0'
```
Comment (BenjaminRodenberg) [inline .github/workflows/action.yml:11]: Shouldn't this be 2.5.0 ?

Comment, not critical: On the develop branch of the Matlab bindings ideally you would always test against the latest version of preCICE (in order to fail early). But I don't know whether this is supported upstream. You might have to build from source for this to work.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        - release: R2020a
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: build Matlab bindings
+        uses: ./.github/workflows/
+        with:
+          matlab-release: ${{ matrix.release }}
+          precice-version: '2.4.0'
```
Comment (BenjaminRodenberg) [inline .github/workflows/test.yml:24]: 2.5.0?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
```
Comment (BenjaminRodenberg) [inline .github/workflows/test.yml:50]: 2.5.0?

File: solverdummy/README.md [hunk 4]
```diff
@@ -1,8 +1,6 @@
 # MATLAB solverdummy
 
-This dummy illustrates the use of precice with MATLAB using the MATLAB bindings based on the C data API.
-
-**Please note:** Currently, the MATLAB solverdummy can't be coupled with other solvers unless an exchange directory is specified with its absolute path in the config file. So, if you wish to couple with a different solver, please make sure to modify the config file accordingly.
```
Comment (BenjaminRodenberg) [inline solverdummy/README.md:5]: You fixed this? Nice! Then maybe also add a changelog entry for this.

**Review by BenjaminRodenberg — COMMENTED** [review]
From my perspective ok as soon as the points below are fixed. Note: I did not try to install or use the matlab bindings.

Suggestion: Since you have to update the preCICE version at multiple places, I would suggest to move to one single point where the preCICE version is stored. Then you only have to update once. You could use an environment variable for this (not sure whether this works globally). Alternatively, use a secret that you define in the repository settings. See https://adamtheautomator.com/github-actions-environment-variables/ for both options.

File: solverdummy/README.md (hunk 4 above)
Comment (erikscheurer) [reply solverdummy/README.md:5]: I couldn't recreate the error described, so as you suggested in #18, this was probably a documentation error.

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    name: Test Matlab solverdummmies
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+        - release: latest
+        - release: R2021b
+        - release: R2021a
+        - release: R2020b
+        - release: R2020a
```
Comment (IshaanDesai) [inline .github/workflows/test.yml:15]: Should we also test for the latest Matlab release [2022a](https://www.mathworks.com/help/matlab/release-notes.html)? Technically this is the latest release of Matlab and we could test on it. This is not a must though.

File: README.md [hunk 6]
```diff
@@ -79,6 +79,25 @@ $ pkg-config --cflags --libs libprecice
...
+To solve this error start matlab with the following command:
+
+```shell
+LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6 matlab
+```
+
+
 # Contributors
 
 * [Dominik Volland](https://github.com/Dominanz) contributed first working prototype in [PR #494 on `precice/precice`](https://github.com/precice/precice/pull/494)
```
Comment (IshaanDesai) [inline README.md:103]: @erikscheurer please also add yourself here :smile: you can write that you added the CI for the bindings and also added new API functions!

**Review by IshaanDesai — COMMENTED** [review]
Two minor comments, otherwise looks good and ready to release.

File: .github/workflows/test.yml (hunk 5 above)
Comment (erikscheurer) [reply .github/workflows/test.yml:15]: This is covered with `release: latest` or do you mean that we should explicitly name it here?

Comment (IshaanDesai) [reply .github/workflows/test.yml:15]: Okay, if it is covered in `release: latest` then it is fine.

**Review by IshaanDesai — APPROVED** [review]

**Pull request merged**